tests/conftest.py

```diff
@@ -38,14 +38,32 @@ def local_server(httpserver) -> LocalHttpServer:
 
 
 @pytest_asyncio.fixture
-async def websocket():
-    """Return a websocket connection to the browser on localhost."""
+async def _websocket_connection():
+    """ Return a websocket connection to the browser on localhost without an
+    active BiDi session.
+    """
     port = os.getenv("PORT", 8080)
     url = f"ws://localhost:{port}"
     async with websockets.connect(url) as connection:
         yield connection
 
 
+@pytest_asyncio.fixture(params=[{"capabilities": {}}])
+async def websocket(request, _websocket_connection):
+    capabilities = request.param['capabilities']
+    """Return a websocket with an active BiDi session."""
```

Comment:
nit: docstring should come first, right after the function definition

Reply:
done